book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
+                 &         &\ e \neq \ell.\\
+                 & =: \lim_{\triangle \tau \to 0} \frac{P(E(\tau + \triangle\tau)=e|E(\tau-) = \ell)}{\triangle \tau}&\nonumber
+\end{aligned}
+$$ {#eq-ms-transition-hazards}
 
-A common multi-state model, the illness-death model, is illustrated in @fig-multi-state, left panel.
-The states are 'healthy' (State 0), with illness (State 1), or death (State 2).
-Healthy subjects can transition directly to death ($0\rightarrow 1$) or after previous illness ($0\rightarrow 1 \rightarrow 2$), without the possibility of back-transitions.
-Similar to competing risks, individual transitions are characterised by transition-specific hazards, denoted by $h_{\ell\rightarrow e}(\tau)$ or short $h_{\ell e}(\tau)$, where $\ell$ is the starting state and $e$ the end state  for a transition, $\ell, e \in \{0,\ldots,k\}$.
-Remaining in the same state (transition into the same sate $h_{\ell\ell}(\tau)$) is equivalent to being censored for any transition possible from that state.
+Transition hazards @eq-ms-transition-hazards indicate the relative risk to enter state $e$ at time $\tau$ given occupation of state $\ell$ in the instant before $\tau$ ($\tau-$).
 
-![Left: The illness-death model where subjects can transition from states: healthy (0), with illness (1), and terminal state death (2) without back-transitions. Right: Illustration of a general multi-state model with back-transitions. Here subjects can transition from state healthy (0) to different disease progressions (1, 2) with (partial) recovery (0, 1) and terminal state death (3).](Figures/survival/multi-state-examples-w-transitions.png){#fig-multi-state fig-alt="Schematic illustration of different multi-state models."}
+Analogous to the competing risks setting, we can define the transition specific cumulative hazards
 
-The right panel of @fig-multi-state depicts a more general multi-state setting with back-transitions.
-One can think of transitions between a healthy state (0) to different disease progressions (1, 2) with possibilty of improvement and recovery, and possible transition to a terminal state (3).
+$$
+H_{\ell e}(\tau) = \int_{0}^{\tau}h_{\ell e}(u)du
+$$ {#eq-ms-cumu-hazard}
 
-When modeling such data, interest often lies in estimation of the transition hazards, but also in transition probabilities $P_{\ell e}(\zeta, \tau)$, that is the probability to transition from state $\ell$ to $e$ between time points $\zeta$ and $\tau$ or state occupation probabilities $P_e(\tau)$, the probability to occupy state $e$ at time $\tau$.
-Transition probabilities are often summarized in a matrix of transtion probabilities $\mathbf{P}(\zeta, \tau)$, where rows indicate starting states and columns indicate end states.
 
-For example, for the illness-death model, the transition matrix would be given as
+As in the competing risks setting, however, (cumulative) transition hazards don't tell the full story when absolute probabilities of transition are of interest.
```

Comment:
Needs a rewrite, saying "as in however" is just confusing, is it the same or not?